Automattic/simplenote-macos
# Fastlane: Localization Step(s)

### Details:
1. **Fastlane:** Wires `ios_localize_project` in the `code_freeze` step
2. **Fastlane:** Adds the `update_appstore_strings` step
3. Updates the `Release Toolkit` dependency

And we're also adding the `AppStoreStrings.pot` file (along with the `subtitle` / `description` / `keywords` text files).

Ref. #8

/cc @mokagio @loremattei 
Thanks in advance gentlemen!!

---

### Test: code_freeze
1. Check out commit 235c8cb
2. **TURN OFF your wifi!!!**
3. Run `bundle exec fastlane code_freeze`

- [ ] Verify there's a new commit in your local repository called `Updates strings for localization` 

* Note: In commit 235c8cb we've got a "hacked" code_freeze version, which only runs `ios_localize_project`!


### Test: update_appstore_strings
1. Check out commit 235c8cb
2. **TURN OFF your wifi!!!**
3. Run `bundle exec fastlane update_appstore_strings version:1.12`

- [ ] Verify there's a new commit in your local repository called `Update metadata strings`
- [ ] Verify the updated `AppStoreStrings.pot` was modified (3 changes!)

**Please** discard this new commit!

### Release
These changes do not require release notes.
